docker/images/pinot-base/pinot-base-runtime/amazoncorretto.dockerfile

```diff
@@ -0,0 +1,40 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+ARG JAVA_VERSION=11
+ARG JDK_IMAGE=amazoncorretto
+
+FROM ${JDK_IMAGE}:${JAVA_VERSION}-al2-jdk
```

Comment:
This is the only image that doesn't derive from `ubuntu`. I don't mind to use alpine images (although I've never like the hype on alpine docker images, the base image should be cached in the node anyway), but having to choose between alpine or ubuntu depending on the vendor doesn't sound good.

I mean, I would prefer to have a dockerfile for ubuntu, a dockerfile for alpine and choose the _meta-base_ (either corretto, ms or whatever) based on docker `ARG`s. Otherwise we will see a combinatorial explosion of different dockerfiles depending on the openjdk distribution and the linux distribution